README.md

```diff
@@ -70,6 +70,18 @@ You can also use this feature to set which item should be open when mounted.
 
 If you need control of the selected item, you can use the `key` prop and the `on:change` event to manage the state of the component externally.
 
+```html
+<script>
+    let currentKey = null;
+    function handleEvent(event){
+        currentKey = event.detail.key;
+    }
+</script>
+
+<Accordion on:change={handleEvent}>
```

Comment:
So, I actually think that you can use two-way binding and then you don't even need on:change. Something like ..
```
<script>
    let currentKey = null
</script>

<Accordion bind:key={currentKey} />
```
Can you check to see if this also works for your use-case?

Reply:
Oh wow, that's work ! sorry for the inconvenience.
I close this.